src/modules/profile/views/waitingTimeStatistics/waitingTimeStatistics.tsx

```diff
@@ -0,0 +1,89 @@
+import { ArcElement, Chart as ChartJS, type ChartOptions, Legend, Tooltip } from 'chart.js';
+import { useEffect, useMemo, useState } from 'react';
+import { Pie } from 'react-chartjs-2';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+interface statistics {
+  waiting_time: number,
+  waiting_time_title: string,
+  waiting_time_count: number,
+  waiting_time_percent: number
+}
+
+interface WaitingTimeStatisticsProps {
+  slug: string;
+  className?: string;
+  statistics: statistics[];
+}
+
+const WaitingTimeStatistics = (props: WaitingTimeStatisticsProps) => {
+  const { className, statistics = [] } = props;
+  const [options, setOptions] = useState<ChartOptions<'pie'>>({});
+
+  const datasets = useMemo(() => {
+    const total = statistics.length
+    return {
+      total: total,
+      data: statistics,
+      values: statistics.map((wt: any) => wt.waiting_time_percent),
+      legendLabels: statistics.map((wt, index) => `%‎${wt.waiting_time_percent.toLocaleString('fa')} ` + wt.waiting_time_title),
+    }
+  }, [statistics]);
+
+  const data = {
+    labels: datasets.legendLabels,
+    datasets: [
+      {
+        data: datasets.values,
+        backgroundColor: ['rgba(57, 146, 61, 1)', 'rgba(58, 180, 64, 1)', 'rgba(255, 173, 13, 1)', 'rgba(240, 115, 0, 1)'],
+        borderColor: 'rgba(255, 255, 255, 1)',
+        borderWidth: 1,
+      },
+    ],
+  };
+
+  useEffect(() => {
+    const style = getComputedStyle(document.body);
+    const font = style.getPropertyValue('--font-iran-sans');
```

Comment:
**Coupling to Global Styles**:
Your component's appearance is directly tied to the global styles. If `--font-iran-sans` is not defined or is changed, it could affect the appearance of your component.

Is this necessary? Can it be without dependency on `--font-iran-sans`?